### PR TITLE
Make outfit

### PR DESCRIPTION
**Pull request by Chrisgood2go**
got card to render in outfit based on style 

got button in outfit to delete cards from list

worked mostly on bug that caused by trying to render because the way it is written now it tries to get related products to update when I press the button  tto render relational modal and it doesn't let me do that.
Im thinking i need some modularization so modal will exist and render out of related products.
so i created another file to do that and have the carousel for related list there its not attached to anything yet. the code being ran is still the carousel in related products.

### CODE REVIEW

File: client/src/components/relatedProducts/outfitList.jsx [hunk 1]
```diff
@@ -66,24 +66,30 @@ const Outfit = (props) => {
...
-                              '$7327.00' : null}
-                          key={product_id}
-                          button={<FontAwesomeIcon icon={faCircleXmark}
+                            '$' + style.sale_price : null}
+                          key={product_id+props.addStyle}
+                          btnStyle={'x'}
+                          handleXClick = {
+                            ()=>{
+                            console.log('X button clicked from outfitlist' )
+                            setList((list)=> list.filter(item => item.key !== product_id+props.addStyle))
```
Comment (Nkozlare) [inline client/src/components/relatedProducts/outfitList.jsx:89]: Console log on line 88

File: client/src/components/relatedProducts/relatedProductsList.jsx [hunk 2]
```diff
@@ -71,37 +80,53 @@ const RelatedProducts = (props) => {
...
               salePrice={style.sale_price ?
-                '$' + style.sale_price : index % 2 === 0 ?
-                  '$7327.00' : null}
+                '$' + style.sale_price : null}
               key={index}
-              button={<StarBtn />}
+              // button={<FontAwesomeIcon  onClick={handleStarClick} style={second} icon={faStar} />}
+              btnStyle={'star'}
+              setIsOpen = {setIsOpen}
               handleChangeProduct ={props.handleChangeProduct}
```
Comment (Nkozlare) [inline client/src/components/relatedProducts/relatedProductsList.jsx:116]: If it is in quotes I don't believe it needs to be in curly braces.

File: client/src/components/relatedProducts/outfitList.jsx (hunk 1 above)
Comment (Chrisgood2go) [reply client/src/components/relatedProducts/outfitList.jsx:89]: will keep for now why building, will only log when clicked.

File: client/src/components/relatedProducts/relatedProductsList.jsx (hunk 2 above)
Comment (Chrisgood2go) [reply client/src/components/relatedProducts/relatedProductsList.jsx:116]: roger, will correct that.

**Review by Nkozlare — APPROVED** [review]
I think your code is formatted pretty well. It is pretty easy to follow. There were a number of console logs that you probably want to get rid of.

**Pull request closed without merging**